sdk/core/azure-core/inc/azure/core/http/win_http_transport.hpp

```diff
@@ -134,6 +134,27 @@ namespace Azure { namespace Core { namespace Http {
     bool IgnoreUnknownCertificateAuthority = false;
```

Comment:
Why is NoClientCertificate not a transport option?

Reply:
If we make it a transport option, customers will need to turn it ON when using Attestation.

We need a way to internally patch the SDK client without a customer noticing it.

We also need to be able to create a transport adapter and use it to create Storage and Attestation clients and it should work for both.